utilities/tools/Set-ModuleReadMe.ps1

```diff
@@ -1475,7 +1477,11 @@ function Set-ModuleReadMe {
     }
 
     $moduleRoot = Split-Path $TemplateFilePath -Parent
-    $fullModuleIdentifier = 'Microsoft.{0}' -f $moduleRoot.Replace('\', '/').split('/Microsoft.')[1]
+    $moduleRelative = $moduleRoot.Replace('\', '/').split('modules/')[1]
```

Comment:
I wonder if we should move this into its own function / script for transparency. Rainer will need a similar logic for his changes in the BRM tool. And I assume we'll anyways need it also in other places later. Thoughts?
We already reference external scripts like in line 1457 ff

Reply:
We can diacuss it, but please not in this PR

Reply:
Wouldn't this PR be exactly the place, as the code was created in this PR? Happy to delay it - just question the motives :D

Reply:
This PR is splitting on the modules folder instead of on the Microsoft. prefix as it was before. Always happy to improve the code, I was questioning if the case to do that in a PR containing more than 1700 file changes.

Reply:
Fair enough - as long as we don't forget about it. You created a neat piece of code here - and I just hope we don't loose it this huge file when we need it again later :) Otherwise I suspect somebody will re-implement a similiar logic with the same goal. 
Feel free to resolve the comment.

Reply:
Created issue #3187